src/sentry/utils/audit.py

```diff
@@ -25,7 +26,7 @@
 
 
 def create_audit_entry(
-    request: HttpRequest,
+    request: HttpRequest | Request,
```

Comment:
hmmm so I'm suspicious of almost every `: Request` in the codebase -- that they usually should be `HttpRequest` -- would it be possible to fix the caller here?

Reply:
oh actually this is passes typing when i remove it 🤔 , thanks for calling out